doc/user/walkthrough/backup-operator.md

```diff
@@ -63,7 +63,7 @@ Create EtcdBackup CR:
 >Note: this example uses S3 Bucket "mybucket" and k8s secret "aws"
 
 ```sh
-sed -e 's/<s3-bucket-name>/mybucket/g' \
+sed -e 's|<full-s3-path>|mybucket/myprefix/2017-12-14T20:33:41Z/etcd.backup|g' \
```

Comment:
can we make the full s3path as simple as possible?
e.g
`mybucket/myprefix/2017-12-14T20:33:41Z/etcd.backup` ->
`mybucket/etcd.backup`?

Reply:
Good suggestion. Fixed!